plugin.yaml

```diff
@@ -3,575 +3,619 @@
-                type: string
-                default: Organization
-            custom_sysprep:
-                description: >
-                    A custom System Preparation Answers file
-                    to use for full customization of Windows.
-                    This can be generated by the Windows System Image Manager.
-                    Note that this file should be verified to work correctly before being applied,
-                    as any errors will appear only on Windows and will not be visible to the plugin.
-                    Also note that any scripts, etc.,
-                    that attempt to work on the VM after the custom sysprep must tolerate multiple retries
-                    because the plugin cannot detect when the custom sysprep has finished,
-                    so provides the server as soon as the IPs are assigned
-                    (which occurs before customization is complete).
-                type: string
-                required: false
-            os_family:
-                default: windows
-                description: Overridden default from ``Server`` node_type.
-            agent_config:
-                type: cloudify.datatypes.AgentConfig
-                default:
-                    port: 5985
-
-    cloudify.vsphere.nodes.Network:
-        derived_from: cloudify.nodes.Network
-        properties:
-            use_existing_resource:
-                description: >
-                    Whether to use a network that already exists on vSphere.
-                default: False
-            network:
-                description: |
-                    key-value network configuration. ::
-
-                        name: network name
-                        vlan_id: vLAN identifier which will be assigned to the network
-                        vswitch_name: name of the vSwitch the network will be connected to.
-            connection_config:
-                default: {}
-                type: cloudify.datatypes.vsphere.Config
-        interfaces:
-            cloudify.interfaces.lifecycle:
-                create:
-                    implementation: vsphere.vsphere_network_plugin.network.create
-                    inputs: {}
-                delete:
-                    implementation: vsphere.vsphere_network_plugin.network.delete
-                    inputs: {}
-
-    cloudify.vsphere.nodes.Storage:
-        derived_from: cloudify.nodes.Volume
-        properties:
-            storage:
-                description: |
-                  key-value storage disk configuration. ::
-
-                    parent_key: Device key from controller, negative mean
-                      that can be used any
-
-                    storage_size: disk size in GB.
+  vsphere:
+    executor: central_deployment_agent
+    package_name: cloudify-vsphere-plugin
+    package_version: '2.8.1'
+    source: https://github.com/cloudify-cosmo/cloudify-vsphere-plugin/archive/2.8.0.zip
```

Comment:
Can you please undo the indentation changes on this file? It's unreviewable in its current form because it shows the entire file as having changed.